src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php

```diff
@@ -116,6 +118,10 @@ protected function createDescriptionAboutText(?int $releaseGroupId): string
             $text .= sprintf(' for [%s](%s) release group', $releaseGroupLink, $releaseGroupLink);
         }
 
+        if ($jiraIssueLink !== null) {
+            $text .= sprintf('.%sJira ticket [%s](%s)', PHP_EOL, $jiraIssueLink, $jiraIssueLink);
```

Comment:
Why it is not in title? In the requirements said "I see Spryker’s Jira Ticket in PR title for the core release. (only for core)". 
The second question is, how to be sure, this one will work only for core integrations?
And the last one. What if there are a few RGs installed?